examples/fluids/Makefile

```diff
@@ -41,6 +41,7 @@ AFLAGS ?= -fsanitize=address #-fsanitize=undefined -fno-omit-frame-pointer
 CFLAGS += $(if $(ASAN),$(AFLAGS))
 FFLAGS += $(if $(ASAN),$(AFLAGS))
 LDFLAGS += $(if $(ASAN),$(AFLAGS))
+CPPFLAGS += -I./include
```

Comment:
I'm pretty meh at Make, but I'm confused as to why this would be only for `CPPFLAGS` and not `CFLAGS`?

Reply:
That... a great question. TBH, I actually just [took the line from Ratel](https://gitlab.com/micromorph/ratel/-/blob/main/Makefile?ref_type=heads#L80).

Reply:
I guess the preprocessor is responsible for handling the include file substitutions, so it's the only one that needs to know about them.

Reply:
Hmm, its in the libCEED main Makefile too so there has to be something new to learn about Make or an old benign bug. Not sure which

Reply:
I think it's probably what I said above; The include files are only needed for preprocessing. They're not used during the compilation or linking phase, so no need to put them there too.

Reply:
Note that `CFLAGS` is never used without `CPPFLAGS`, but there are times we need only `CPPFLAGS`.
```makefile
$(OBJDIR)/%.o : $(CURDIR)/%.c | $$(@D)/.DIR
	$(call quiet,CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $(abspath $<)
```
I suppose if we stored the result of preprocessing, it would technically be possible to compile without `CPPFLAGS`.